docs/rst/manual/reference/queue.rst

```diff
@@ -1,12 +1,154 @@
-Queue System (todo)
+Queue System
 ===================
 
+The Queue System is responsible for scheduling, running, monitoring and acting
+on a set of realizations.
+
 Available queue systems
 -----------------------
-.. todo::
-   List of all available queue systems and how to configure them
 
-Parallel
----------
-.. todo::
-   Parallel
+A number of queue systems are pre-installed with ERT.
+
+- ``LOCAL``
+- ``LSF``
+- ``TORQUE``
+- ``RSH`` — `deprecated`
+
+The ``LOCAL`` queue system will, unsurprisingly, run all computations locally,
+whereas the remaining systems requires access to some external resource. Let's
+first configure a local queue system.
+
+Local
+-----
+
+Available local configuration options
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block::
```

Comment:
I think the reason why it doesn't build is because you need to specify a language in the code block? Interesting that it passes in python 3 but not in python 2...

Reply:
https://www.sphinx-doc.org/en/1.5/markup/code.html